lib/feature_flagger/storage/redis.rb

```diff
@@ -50,6 +71,32 @@ def all_values(key)
       def search_keys(query)
         @redis.scan_each(match: query)
       end
+
+      private
+
+      def resource_key(resource_name, resource_id)
+        FeatureFlagger::Storage::RedisKeys.resource_key(
+          RESOURCE_PREFIX,
+          resource_name,
+          resource_id,
+        )
+      end
+
+      def remove_feature_key_from_resources(feature_key)
+        cursor = 0
+
+        loop do
+          cursor, resource_keys = @redis.scan(cursor, match: "#{RESOURCE_PREFIX}:*", count: SCAN_EACH_BATCH_SIZE)
```

Comment:
Is it possible to use SSCAN here? Looks like we're only interested in SETs https://redis.io/commands/sscan

Reply:
Nevermind, SSCAN only iterates on a single SET element.

Reply:
`SSCAN` performs scan in a SET, what we want actually is to perform a scan in the keyspace.

```
127.0.0.1:6379> SADD mytestset M1 M2 M3 N1 N2 N3 O1 O2 O3 P1 P2 P3 Q1 Q2 Q3

127.0.0.1:6379> SSCAN mytestset 0 MATCH N*
1) "3"
2) 1) "N1"
   2) "N3"
   3) "N2"
127.0.0.1:6379> SSCAN mytestset 0 MATCH *3*
1) "3"
2) 1) "Q3"
   2) "O3"
   3) "N3"
127.0.0.1:6379> SSCAN mytestset 0 MATCH *3* COUNT 20
1) "0"
2) 1) "Q3"
   2) "O3"
   3) "N3"
   4) "P3"
   5) "M3"
```